files/entrypoint.sh

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env bash
+
+
+# Variables
+# --
+#
+# These values can be overridden by ENV Vars
+#
+K8S_VERSION=${K8S_VERSION:-v1.6.1}
+KUBEADM_RELEASE=${KUBEADM_RELEASE:-v1.6.0-alpha.0.2074+a092d8e0f95f52}
```

Comment:
Yes I know this is the version I am currently using :)

But since this can be overridden by ENV variables at `docker run` I thought it was not necessary to change it once again... I only merged #7 at first before submitting the PR.